src/howler.core.js

```diff
@@ -1910,10 +1910,10 @@
     _cleanBuffer: function(node) {
       var self = this;
```

Comment:
you could have just changed `var self = Howler` instead of self in two other spots